src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -856,17 +856,14 @@ enum Mode {
     FERRY
 
     """Enables flexible transit for access and egress legs"""
-    FLEXIBLE
+    FLEX
```

Comment:
The name of the enum changes here so that the generated code doesn't need to be modified.

(The enum handling still needs a few manual changes after generation)